File: 7-kyu/sum-of-two-lowest-positive-integers/c++/solution.cpp
#include <algorithm>

long sumTwoSmallestNumbers(std::vector<int> numbers)
{
    long min = *(std::min_element(numbers.begin(), numbers.end()));
    long next_min = *(std::max_element(numbers.begin(), numbers.end()));
    for (const int n : numbers) {
        if (n < next_min && n > min) next_min = n;
    }
    return min + next_min;
}

File: 7-kyu/sum-of-two-lowest-positive-integers/javascript/solution.js
func sumOfTwoSmallestIntegersIn(_ array: [Int]) -> Int {
  let min = array.min()!
  var nextMin = array.max()!
  for n in array {
    if (n < nextMin && n > min) {
      nextMin = n
    }
  }
  return min + nextMin
}